_SQLServer/TableLoadHeroes.sql

```diff
@@ -0,0 +1,478 @@
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Ember_Mage_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Ember_Mage.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'The soul of the mage will reflect the powers that they develop.  Ember Mages are quick to anger, and wield terrible blasts of fire and brimstone.  Their destructive magic is highly sought after by adventuring parties willing to take the risk.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Fire Wave
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Magma Strike
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --White Mage Potion
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --Fire Water
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Human Hero', 6, 3, '2B', '3B', '2R', '1B1R', 5, 2);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,3,8);--! To look at later
+insert into DefenseAssignments (CharacterIndex, AttributeIndex) VALUES (0,2);--! To look at later
+
+
+--Glimmerdusk Ranger
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Glimmerdusk Ranger', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Glimmerdusk_Ranger_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Glimmerdusk_Ranger_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Glimmerdusk_Ranger.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'Deadly archers, the Glimmerdusk Rangers patrol edges of the Fae Wood hunting any foolish enough to enter uninvited.  With the rise of the Consul and the Nether Rifts advancing into their lands, the Rangers have begun to venture from their woodland realm.  They offer their considerable skill to any who will take the fight to the Consul.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Herbalist
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Sparkle Burst
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --Pixie Dust
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Elf Hero', 7, 3, '1B1R', '3B', '2B', '2R', 5, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,4,8);--! To look at later
```

Comment:
This character should have a 1st offense assignment for attack (0,0,1)

Reply:
Completed - see line above